client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -119,72 +107,95 @@ module.exports = React.createClass( {
 			}
 		];
 
-		const summaryPageSlug = this.props.site ? this.props.site.slug : '';
-		if ( 'email-followers' === activeFilter ) {
-			summaryPageLink = '/stats/follows/email/' + summaryPageSlug;
-		} else {
-			summaryPageLink = '/stats/follows/wpcom/' + summaryPageSlug;
-		}
-
-		if ( wpcomData && wpcomData.subscribers ) {
-			wpcomFollowers = <StatsList moduleName="wpcomFollowers" data={ wpcomData.subscribers } followList={ this.props.followList } />;
-		}
-
-		if ( emailData && emailData.subscribers ) {
-			emailFollowers = <StatsList moduleName="EmailFollowers" data={ emailData.subscribers } />;
-		}
-
-		if ( wpcomData && wpcomData.total ) {
-			wpcomTotalFollowers = <p>{ this.translate( 'Total WordPress.com Followers' ) }: { this.numberFormat( wpcomData.total ) }</p>;
-		}
-
-		if ( emailData && emailData.total ) {
-			emailTotalFollowers = <p>{ this.translate( 'Total Email Followers' ) }: { this.numberFormat( emailData.total ) }</p>;
-		}
-
-		if ( ( wpcomData && wpcomData.viewAll ) || ( emailData && emailData.viewAll ) ) {
-			viewSummary = (
-				<div key="view-all" className="module-expand">
-					<a href={ summaryPageLink }>{ this.translate( 'View All', { context: 'Stats: Button label to expand a panel' } ) }<span className="right"></span></a>
-				</div>
-				);
-		}
+		const summaryPageSlug = siteSlug || '';
+		const summaryPageLink = 'email-followers' === activeFilter
+			? '/stats/follows/email/' + summaryPageSlug
```

Comment:
I think in a follow-up PR we should link to `/people/followers/${ slug }` and `/people/email-followers/${ slug }`.  Prior to doing so we need to add a download csv link to the email-followers.